lib/src/components/NxTransferList/NxTransferList.scss

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-text-helpers';
+
+// the padding on the sides of each transfer list control box
+$nx-transfer-list-control-box-side-spacing: var(--nx-spacing-6x);
+$nx-transfer-list-item-vertical-padding: var(--nx-spacing-2x);
+$nx-transfer-list-item-vertical-margin: var(--nx-spacing-2x);
+$nx-transfer-list-visible-item-count: 10;
```

Comment:
Would this better as a CSS variable? In case a consuming project needs/wants to make the columns taller for some reason?

Reply:
> Would this better as a CSS variable? In case a consuming project needs/wants to make the columns taller for some reason?

That's only useful if they want to make them taller but still define their height in terms of a number of visible items.  If they just want to make it some number of pixels tall they'd use a normal CSS override style.  So I think it's too much of an edge case to worry about given that no one's asking for it.  Keep in mind if we did that we'd have to go through the trouble of documenting it, worrying about backwards compatibility for it, etc.